src/deluge/gui/views/view.cpp

```diff
@@ -1354,6 +1359,15 @@ void View::drawOutputNameFromDetails(InstrumentType instrumentType, int32_t chan
 	}
 
 	if (display->haveOLED()) {
```

Comment:
when the automation instrument clip view is loaded or you change instrument types within the view the screen is re-rendered. Added this code to render the new OLED UI for the Automation Instrument Clip View.